Library/Formula/scriptcs.rb

```diff
@@ -0,0 +1,25 @@
+require "formula"
+
+class Scriptcs < Formula
+  homepage "https://github.com/scriptcs/scriptcs"
+  url "https://github.com/scriptcs/scriptcs.git", :tag => "v0.13.3"
+  depends_on "mono" => :recommended
```

Comment:
Add another blank line before the deps. They don't get squashed into the url/home block.